.github/workflows/release.yml

```diff
@@ -66,6 +66,48 @@ jobs:
         uses: actions/upload-pages-artifact@56afc609e74202658d3ffba0e8f6dda462b719fa # v3.0.1
         with:
           path: apps/strapi.frameless.io/build/
+      - name: Start Deploy Tiptap Sandbox Website to Vercel
+        uses: bobheadxi/deployments@648679e8e4915b27893bd7dbc35cb504dc915bc8 # v1.5.0
+        id: deployment-preview
+        with:
+          step: start
+          token: ${{ secrets.GITHUB_TOKEN }}
+          env: Preview
+      - name: Deploy Tiptap Sandbox Website to Vercel/Preview
+        id: vercel-action-preview
+        if: github.ref != 'refs/heads/main'
+        uses: amondnet/vercel-action@16e87c0a08142b0d0d33b76aeaf20823c381b9b9 # v25.2.0
+        with:
+          vercel-token: ${{ secrets.VERCEL_TOKEN }}
+          vercel-org-id: ${{ secrets.VERCEL_ORG_ID }}
+          vercel-project-id: ${{ secrets.VERCEL_PROJECT_ID }}
+          scope: ${{ secrets.VERCEL_ORG_ID }}
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          working-directory: ./apps/tiptap-sandbox
+      - name: Deploy Tiptap Sandbox Website to Vercel/Production
+        uses: amondnet/vercel-action@16e87c0a08142b0d0d33b76aeaf20823c381b9b9 # v25.2.0
+        id: vercel-action-production
+        if: github.event_name == 'push' && github.ref == 'refs/heads/main'
+        with:
+          vercel-token: ${{ secrets.VERCEL_TOKEN }}
+          vercel-org-id: ${{ secrets.VERCEL_ORG_ID }}
+          vercel-project-id: ${{ secrets.VERCEL_PROJECT_ID }}
+          scope: ${{ secrets.VERCEL_ORG_ID }}
+          github-comment: false
+          vercel-args: "--prod"
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          working-directory: ./apps/tiptap-sandbox
```

Comment:
Ik vraag me af of we al die pakketjes wel moeten vertrouwen, zoals:
- `amondnet/vercel-action`
- `bobheadxi/deployments`

Misschien kunnen we wel beter de officiele Terraform voor Vercel gebruiken:

https://github.com/vercel/terraform-provider-vercel